src/dune

```diff
@@ -0,0 +1,5 @@
+(library
+  (name enumerators)
+  (public_name enumerators)
```

Comment:
You can also remove `(name)` if it's equal to the `(public_name)` (requires lang 1.1)

Reply:
also it's `wrapped false` so `name` actually doesn't matter here